app.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "rogue",
+  "description": ":sparkles:",
+  "env": {
+    "APP_ENV": "local",
+    "APP_DEBUG": "true",
```

Comment:
@blisteringherb Thanks for fixing this in production, let's make sure we change these here too before merging.

```diff
-    "APP_ENV": "local",
-    "APP_DEBUG": "true",
+    "APP_ENV": { "required": true },
+    "APP_DEBUG": "false",
```